docs/development_tips.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Tips and Tricks"
+---
+
+This page provides some tips and tricks that may help you as a contributor to Home Assistant. The list here is by no means exhaustive, and if you pick up any additional undocumented tips and tricks, please open a PR to add them here.
+
+## Tips and Tricks
+
+### Keep PRs simple
+
+It's in your best interest to keep your PRs as short and simple as possible. It may feel like extra work on your part to split a change you have in mind into smaller changes, but on the plus side:
```

Comment:
Looks like this is similar to https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=small#5-make-your-pull-request-as-small-as-possible ?

Reply:
well on the plus side, this PR has been exposing me to areas of the dev docs that I have missed. I will link to that here as well 😅

Reply:
I thought about this some more and I think there's valuable information in this text that's not included there. That is more of a policy, this is more of a guidance, but maybe I am over thinking this?

Reply:
OK I've taken the pieces that I think would be useful and moved them into that linked page